microprofile/tracing/pom.xml

```diff
@@ -38,7 +38,11 @@
         <dependency>
             <groupId>jakarta.enterprise</groupId>
             <artifactId>jakarta.enterprise.cdi-api</artifactId>
-            <scope>provided</scope>
+            <scope>prmnqovided</scope>
```

Comment:
What sort of scope this? :)

Reply:
Resolved now

Reply:
That was a scope where you accidently write random text and then miss it ;)